src/test/bin/winkernel/control.cpp

```diff
@@ -485,6 +485,8 @@ size_t QUIC_IOCTL_BUFFER_SIZES[] =
     sizeof(QUIC_RUN_FEATURE_NEGOTIATION),
     sizeof(QUIC_RUN_FEATURE_NEGOTIATION),
     0,
+    0,
+    0
```

Comment:
```suggestion
    0,
```
Always leave a trailing `,` to make the diff nicer when you add a new line (don't have to modify the old one)